examples/grpc_sink/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/talos-systems/os-engine/api/types"
+	"google.golang.org/grpc"
+)
+
+type server struct{}
+
+func (s *server) Events(stream types.Sink_EventsServer) error {
```

Comment:
Needs comments to document that this implements the higher level gRPC interface, while the gRPC sink loader implements the internal sink interface.